example-deployment/keycloak-tools/bootstrap-keycloak.js

```diff
@@ -31,9 +31,9 @@ function authHeaders(token) {
       },
       {
         clientId: 'dashboard',
-        rootUrl: 'https://example.com',
-        redirectUris: ['https://example.com/*'],
-        webOrigins: ['https://example.com'],
+        rootUrl: 'https://openrobotics.demo.open-rmf.org',
```

Comment:
we should stick with example.com for the example, you may add a cli arg to change the url, but make example.com the default.

Reply:
Handled here: https://github.com/open-rmf/rmf-web/pull/350/commits/5db60f19a81a84dfcf477083995c230e24dd3e90